dev-libs/libbase58/libbase58-0.1.4.ebuild

```diff
@@ -11,18 +11,37 @@ LICENSE="MIT"
 SRC_URI="https://github.com/luke-jr/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
 SLOT="0/0"
 KEYWORDS="amd64 ~arm ~mips ~ppc ~ppc64 x86"
-IUSE="tools"
+IUSE="test tools"
```

Comment:
I can't get the tests to work, 
```
Testsuite summary for libbase58 0.1.4
============================================================================
# TOTAL: 12
# PASS:  6
# SKIP:  0
# XFAIL: 0
# FAIL:  6
# XPASS: 0
# ERROR: 0
============================================================================
See ./test-suite.log
Please report to luke_libbase58@dashjr.org
============================================================================
make[2]: *** [Makefile:902: test-suite.log] Error 1
make[2]: Leaving directory '/var/tmp/portage/dev-libs/libbase58-0.1.4/work/libbase58-0.1.4-abi_x86_64.amd64'
make[1]: *** [Makefile:1010: check-TESTS] Error 2
make[1]: Leaving directory '/var/tmp/portage/dev-libs/libbase58-0.1.4/work/libbase58-0.1.4-abi_x86_64.amd64'
make: *** [Makefile:1214: check-am] Error 2
 * ERROR: dev-libs/libbase58-0.1.4::testworld failed (test phase):
 *   Make check failed. See above for details.
```
```
=======================================
   libbase58 0.1.4: ./test-suite.log
=======================================

# TOTAL: 12
# PASS:  6
# SKIP:  0
# XFAIL: 0
# FAIL:  6
# XPASS: 0
# ERROR: 0

.. contents:: :depth: 2

FAIL: tests/decode
==================

/var/tmp/portage/dev-libs/libbase58-0.1.4/work/libbase58-0.1.4/tests/decode.sh: line 2: base58: command not found
FAIL tests/decode.sh (exit status: 1)

FAIL: tests/decode-small
========================

/var/tmp/portage/dev-libs/libbase58-0.1.4/work/libbase58-0.1.4/tests/decode-small.sh: line 2: base58: command not found
FAIL tests/decode-small.sh (exit status: 1)

FAIL: tests/decode-zero
=======================

/var/tmp/portage/dev-libs/libbase58-0.1.4/work/libbase58-0.1.4/tests/decode-zero.sh: line 2: base58: command not found
FAIL tests/decode-zero.sh (exit status: 1)

FAIL: tests/encode
==================

/var/tmp/portage/dev-libs/libbase58-0.1.4/work/libbase58-0.1.4/tests/encode.sh: line 2: base58: command not found
FAIL tests/encode.sh (exit status: 1)

FAIL: tests/encode-b58c
=======================

/var/tmp/portage/dev-libs/libbase58-0.1.4/work/libbase58-0.1.4/tests/encode-b58c.sh: line 2: base58: command not found
FAIL tests/encode-b58c.sh (exit status: 1)

FAIL: tests/encode-small
========================

/var/tmp/portage/dev-libs/libbase58-0.1.4/work/libbase58-0.1.4/tests/encode-small.sh: line 2: base58: command not found
FAIL tests/encode-small.sh (exit status: 1)
```

If the binary is gotten from this product (although why would it be a library then?) you should be able to edit the PATH variable including correct BUILD_DIR/something there.